bin/main.rb

```diff
@@ -1,73 +1,73 @@
-class Player
-  attr_accessor :name, :weapon
+require '../lib/player.rb'
+require '../lib/board.rb'
+require '../lib/game.rb'
 
-  def initialize(name, weapon)
-    @name = name
-    @weapon = weapon
-  end
+def draw_board(board)
+  puts '  ¹     | ²     | ³     '
+  puts '        |       |       '
+  puts "    #{board.sq_val[0]}   |   #{board.sq_val[1]}   |   #{board.sq_val[2]}   "
+  puts '        |       |       '
+  puts '        |       |       '
+  puts ' -----------------------'
+  puts '  ⁴     | ⁵     | ⁶     '
+  puts '        |       |       '
+  puts "    #{board.sq_val[3]}   |   #{board.sq_val[4]}   |   #{board.sq_val[5]}   "
+  puts '        |       |       '
+  puts '        |       |       '
+  puts ' -----------------------'
+  puts '  ⁷     | ⁸     | ⁹     '
+  puts '        |       |       '
+  puts "    #{board.sq_val[6]}   |   #{board.sq_val[7]}   |   #{board.sq_val[8]}   "
+  puts '        |       |       '
+  puts '        |       |       '
 end
 
-def draw_board(square_values)
-  puts "     0       1       2   "
-  puts "         |       |       "
-  puts "0    #{square_values[0]}   |   #{square_values[1]}   |   #{square_values[2]}   "
-  puts "         |       |       "
-  puts "  -----------------------"
-  puts "         |       |       "
-  puts "1    #{square_values[3]}   |   #{square_values[4]}   |   #{square_values[5]}   "
-  puts "         |       |       "
-  puts "  -----------------------"
-  puts "         |       |       "
-  puts "2    #{square_values[6]}   |   #{square_values[7]}   |   #{square_values[8]}   "
-  puts "         |       |       "
-end
+print 'Player one name: '
+name = gets.chomp
+player_one = Player.new(name, 'X')
 
-def get_name(num, weapon)
-  puts "Player #{num} name:"
-  name = gets.chomp
-  player_one = Player.new(name, weapon)
-end
+print 'Player two name: '
+name = gets.chomp
+player_two = Player.new(name, 'O')
 
-def switch_player(active_player, player_one, player_two)
-  if active_player == player_one.name
-    active_player = player_two.name
-    active_weapon = player_two.weapon
-  elsif active_player == player_two.name
-    active_player = player_one.name
-    active_weapon = player_one.weapon
-  end
-end
+board = Board.new
```

Comment:
The `Board` class is not the main class of that project. it is inside the `Game` class. 
Please instantiate only the `Game` class into `lib/main.rb` and `board = Board.new` into the `Game` class 

Your code looks very good and you won't make a lot of changes to make it work after that change